shell/platform/darwin/ios/framework/Source/FlutterOverlayView.mm

```diff
@@ -62,6 +62,9 @@ - (instancetype)initWithContentsScale:(CGFloat)contentsScale
     if (pixelFormat == MTLPixelFormatRGBA16Float) {
       self->_colorSpaceRef = fml::CFRef(CGColorSpaceCreateWithName(kCGColorSpaceExtendedSRGB));
       layer.colorspace = self->_colorSpaceRef;
+    } else if (pixelFormat == MTLPixelFormatBGR10_XR) {
+      self->_colorSpaceRef = fml::CFRef(CGColorSpaceCreateWithName(kCGColorSpaceExtendedSRGB));
```

Comment:
Notes from chat: I'm not sure how Metal interprets this combination, extended srgb with BGRA_XR.  Specifically what does a red channel with the 0x3ff value interpreted as?  If it is 1.0, we can't represent wide gamut colors.  If it's 1.0931, it will work since it's the max for displayp3, but does that mean all channels are clamped to that value?  That would be a problem, or are they all interpreted different for each channel?

What we'd need for this to work is for 0x3ff to be interpreted as the maximum value for displayp3 for each channel (it's different for each one [1.09, 1.01, 1.04, 1.0]).  We aren't communicating displayp3 anywhere so that's unlikely the case.

If I had to guess they have specified some arbitrary cut off of 0x3ff to represent that can handle most (all?) possible colorspaces, same thing for interpreting zero.

Reply:
Something like, [0x200, 0x20, 0x20, 0x3ff]  = srgb(1, 0, 0, 1)?  Do they treat alpha different, where 0x3ff is one?

We also have to consider that BGRA10XR isn't available on android.  We'll need f16 support at some point anyway.

Reply:
We're not going to do wide gamut on Android this year at least, it shouldn't be a consideration.